lib/views.js

```diff
@@ -269,12 +269,13 @@ exports.places_by_type_parent_id_name = {
 exports.contacts_by_depth = {
   map: function(doc) {
     if (['person', 'clinic', 'health_center', 'district_hospital'].indexOf(doc.type) !== -1) {
+      var value = doc.patient_id || doc.place_id;
```

Comment:
Yeah, I don't think an object can be both a place and a person. I thought about emitting an array but it should not happen...